src/status_im/contexts/chat/group/details/view.cljs

```diff
@@ -26,27 +27,40 @@
       (rf/dispatch [:deselect-member public-key true]))))
 
 (defn add-member-contact-item-render
-  [{:keys [public-key] :as item} _ _ {:keys [group admin? current-pk]}]
-  (let [{:keys [contacts]} group
-        member?            (contains? contacts public-key)
-        checked?           (reagent/atom member?)]
-    (if (or (= current-pk public-key) (and (not admin?) member?))
-      (fn []
-        [contact-list-item/contact-list-item
-         {:disabled? true
-          :accessory {:disabled? true
-                      :type      :checkbox
-                      :checked?  @checked?}}
-         item])
-      (fn []
-        (let [on-toggle #(group-chat-member-toggle member? (swap! checked? not) public-key)]
-          [contact-list-item/contact-list-item
-           {:on-press                on-toggle
-            :allow-multiple-presses? true
-            :accessory               {:type     :checkbox
-                                      :checked? @checked?
-                                      :on-check on-toggle}}
-           item])))))
+  [{:keys [public-key] :as item} _ _ {:keys [group admin? current-pk chat-id set-has-error]}]
+  (let [{:keys [contacts]}      group
+        member?                 (contains? contacts public-key)
+        selected-contacts-count (rf/sub [:group-chat/manage-members-count chat-id])
+        [checked? set-checked]  (rn/use-state member?)
+        disabled?               (or (= current-pk public-key) (and (not admin?) member?))
+        on-toggle               (rn/use-callback
+                                 (fn [value]
+                                   (set-has-error false)
```

Comment:
better to move this logic to event ns, and use subscriptions for error state

Reply:
Done